lib/ukstore/include/uk/store.h

```diff
@@ -45,9 +45,41 @@ extern "C" {
 #endif
 
 #if !CONFIG_LIBUKSTORE
-
 #define uk_store_libid(libname) (-1)
 
+#define __UK_STORE_STATIC_ENTRY(entry, lib_str, e_type, e_get, e_set)	\
+	static const struct uk_store_entry				\
+	__used __section(".uk_store_lib_" lib_str) __align8		\
```

Comment:
If the UK_STORE library is not there, you can still do this but mark it as __unused and do not put it in a special section.